assets/stylesheets/_index.sass

```diff
@@ -37,6 +37,11 @@ a
   color: $text-color-dark
   padding: 50px
 
+  .sorting
```

Comment:
I usually add css so the page doesn't look horrible... I'm a disaster at css, so any alternative you have will surely be better than mine :-)